src/hooks/usePersonAttributes.tsx

```diff
@@ -0,0 +1,30 @@
+import { openmrsFetch, type PersonAttribute, restBaseUrl } from '@openmrs/esm-framework';
+import { useEffect, useState } from 'react';
+
+export const usePersonAttributes = (patientUuid: string) => {
+  const [personAttributes, setPersonAttributes] = useState<Array<PersonAttribute>>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (patientUuid) {
```

Comment:
Can you conditionally load attributes only for forms that support them? (Here is an example https://github.com/openmrs/openmrs-esm-form-engine-lib/blob/main/src/hooks/usePatientPrograms.ts#L12)